lib/minitest.rb

```diff
@@ -45,6 +45,12 @@ module Minitest
   mc.send :attr_accessor, :info_signal
   self.info_signal = "INFO"
 
+  SUITE_ORDERINGS = [:sorted, :random]
+
+  def self.suite_order
+    @@suite_order || :random
```

Comment:
It's probably better to return `nil` than the wrong value.  For example, someone may have specified "ordered" on the command line, but any method calling this method will assume that the test ordering is random until `run` is called.  If the method returns `nil`, at least callers will know that "it hasn't been set yet".  IOW, right now callers can't tell the difference between "someone specified `random`" and "test ordering hasn't been set yet".